core/src/test/java/org/infinispan/api/mvcc/repeatable_read/WriteSkewTest.java

```diff
@@ -57,6 +57,7 @@ public void setUp() {
       // The default cache is NOT write skew enabled.
       cacheManager = TestCacheManagerFactory.createCacheManager(configurationBuilder);
       configurationBuilder.locking().isolationLevel(IsolationLevel.REPEATABLE_READ);
+      configurationBuilder.clustering().hash().groups().enabled();
```

Comment:
`testDontFailOnImmediateRemovalOfAtomicMaps` uses fine grained maps, so grouping is necessary. Please use `localAddress` in the trace log instead, and drop the `getAddress()` method.